src/Service.Host/client/src/components/reportOptions/EanCodesOptions.js

```diff
@@ -30,38 +41,32 @@ class EanCodesOptions extends Component {
     }
 
     render() {
+        const {classes } = this.props;
         return (
-            <div className="container">
-                <Grid>
-                    <PageHeader>
-                        <small>Ean Codes of Sales Articles Report Options</small>
-                    </PageHeader>
-
-                    <Row>
-                        <Col xs={10}>
-                            <Checkbox checked={this.state.includePhasedOut} onChange={ch => this.handlePhaseOutChange(ch.target.checked)}>
-                                Include phased out
-                            </Checkbox>
-                        </Col>
-                    </Row>
-                    <Row>
-                        <Col xs={10}>
-                            <Checkbox checked={this.state.cartonisedOnly} onChange={ch => this.handleCartonisedOnlyChange(ch.target.checked)}>
-                                Cartonised Only
-                            </Checkbox>
-                        </Col>
-                    </Row>
-                    <Row>
-                        <Col xs={10}>
-                            <Button bsClass="btn" onClick={() => this.handleClick()}>
+            <div className={classes.root}>
+                <Grid container spacing={24} justify="center">
+                    <Grid item xs={12}>
+                    <h4>Ean Codes of Sales Articles Report Options</h4>
+                         <CheckboxWithLabel
+                                label='Include Phased Out'
+                                checked={this.state.includePhasedOut}
+                                onChange={ch => this.handlePhaseOutChange(ch.target.checked)}
```

Comment:
I think the convention for these is to use 'e' for event rather than 'ch' or another symbol